index.js

```diff
@@ -136,8 +132,8 @@ class Erik {
         },
 
         files: [
-          `${this._erikPath}/bundled-deps.js`,
-          this._bundledSpecPath,
+          `${this._erikPath}/remote-deps.js`,
```

Comment:
Will Karma handle this file not existing [if there were no remote dependencies](https://github.com/mixmaxhq/erik/pull/6/files#diff-168726dbe96b3ce427e7fedce31bb0bcR102) to bundle?